src/python/model/README.md

```diff
@@ -0,0 +1,68 @@
+# Qodana imitation model 
+
+## Architecture 
+`RobertaForSequenceClassification` model with `BCEWithLogitsLoss` solve multilabel classification task. 
+
+Model outputs is a tensor of size: `batch_size` x `num_classes`. Class probabilities are recieved by taking `sigmoid` and final predictions are computed by comparing the probability of each class with the `threshold`. 
+
+As classes might be unbalanced the used metric is `f1-score`.
+## What it does
+
+Model has two use cases:
+- It can be trained to predict a unique number of errors in a **block** of code, unfixed length. 
+- It can be trained to predict a unique number of errors in a **line** of code. 
+___
+## Data preprocessing
+
+Please address to the [`following documentation`](https://github.com/hyperskill/hyperstyle/tree/roberta-model/src/python/evaluation/qodana) for labeling dataset and to the [`following documentation`](https://github.com/hyperskill/hyperstyle/tree/roberta-model/src/python/model/preprocessing) to preprocess data for model training and evaluation afterwards. 
+
+___
+
+## How to train model
+
+Run [`train.py`](https://github.com/hyperskill/hyperstyle/blob/roberta-model/src/python/model/train.py) script from the command line with the following arguments:
+
+Required arguments:
+
+`train_dataset_path`, `val_dataset_path` &#8209; path to the `train.csv` and `val.csv` dataset received by running [`split_dataset.py`](https://github.com/hyperskill/hyperstyle/blob/roberta-model/src/python/model/preprocessing/split_dataset.py) script.
```

Comment:
Could you please add some description what are `train.csv` and `val.csv` here or into split_dataset.py directly